<?xml version="1.0"?>
<launch>
  
  <arg name="gazebo" default="true"/>
  <arg name="controller_layout" default="1"/>
  <arg name="map_file" default="/home/lennart/catkin_ws/src/rtc_project/maps/default_map"/>

  <!-- Gazeboo; wird nur gestartet, wenn gazebo == true -->
  <group if="$(arg gazebo)">
    <include file="$(find turtlebot3_gazebo)/launch/turtlebot3_house.launch" />
  </group>
  
  <!-- ds4_driver starten -->
  <include file="$(find ds4_driver)/launch/ds4_driver.launch" /> 

  <!-- ps4_turtle_control starten -->
  <node pkg="rtc_project" 
        type="ps4_slam.py" 
        name="ps4_slam" 
        output="screen" 
        args="$(arg controller_layout) $(arg map_file)" >
        <rosparam command="load" file="$(find rtc_project)/config/ps4_turtle_control.yaml" />
  </node>
  

  <include file="$(find turtlebot3_slam)/launch/turtlebot3_slam.launch" />


</launch>
